rotary/rules/rotary.smk

```diff
@@ -1055,7 +1055,7 @@ rule run_dfast:
 # TODO - add option to control whether --dbmem flag is set (uses more RAM but does faster analysis)
 rule run_eggnog:
     input:
-        protein="{sample}/annotation/dfast/protein.faa",
+        protein="{sample}/annotation/dfast/{sample}_protein.faa",
         install_finished=os.path.join(DB_DIR_PATH,"checkpoints","eggnog_" + VERSION_EGGNOG)
     output:
         eggnog_annotations="{sample}/annotation/eggnog/eggnog.emapper.annotations",
```

Comment:
Will adding `--output [sample_id]` change the name of this output file, e.g., to `[sample_id].emapper.annotations`?

Reply:
@jmtsuji The output flag adds a prefix to all the annotation output files.

``` 
--output FILE_PREFIX, -o FILE_PREFIX
                        base name for output files
```

Reply:
I will see if it causes the `emapper.annotations` to fail.

Reply:
So `emapper.annotations` now has to be proceeded by `{sample}`

Reply:
Solved.